ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.17.yaml

```diff
@@ -79,6 +79,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-custom-cidr-f28
+  cron: 3 10 2 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CLUSTER_NETWORK_CIDR: 10.128.0.0/21
+      CLUSTER_NETWORK_HOST_PREFIX: "23"
+      CLUSTER_NETWORK_SEC_CIDR: 20.129.0.0/20
+      CLUSTER_NETWORK_SEC_HOST_PREFIX: "23"
+    test:
+    - ref: cucushift-upgrade-setedge-2hops
+    - chain: openshift-upgrade-qe-test-chainupgrade
```

Comment:
```suggestion
    - chain: openshift-upgrade-qe-test-chainupgrade
    - ref: cucushift-installer-check-info
```

Reply:
Actually Jamo 's suggestion is better. It was removed but for this case it makes sense to have it.